src/client/src/appStyles.module.css

```diff
@@ -13,15 +13,35 @@
   background-color: var(--vscode-editor-background);
   height: calc(100vh - 124px);
   justify-content: flex-start;
+  position: relative;
+}
+
+.centerViewAzurePage {
+  padding: 0;
 }
 
 .centerViewMaxHeight {
   height: calc(100vh - 51px);
   margin-bottom: 0px;
 }
 
+.optionalFlag {
+  position: absolute;
+  width: 100%;
+  background-color: #0067b8;
+  color: white;
```

Comment:
Can we use vscode theme variables here? I think white is editor-foreground

Reply:
hello jason

Reply:
I actually didn't use a variable because I needed the colours to be the same in both light and dark mode, but you're right it's generally better practice :)  I found a variable that stays white in all mode so I'm gonna use that instead